src/lib/pages/frontsite/ErrorsPage/ErrorsPage.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import { InternalServerError, NotFound } from '@components/HttpErrors';
+
+export class ErrorsPage extends Component {
+  render() {
+    const params = window.history.state ? window.history.state.state : null;
+    if (window.history.state) {
+      console.log(window.history.state.state);
+    }
+
+    if (params) {
+      if (params.errorCode === 429 || params.errorCode === 500) {
+        if (params.errorId) {
+          return <InternalServerError errorId={params.errorId} />;
```

Comment:
I will be the naming police here again, but the name `<InternalServerError>` as a name refers only to 500, so I wouldn't name the component which handles more errors like this. It will be misleading.  I would have a separate component either to handle each code separately, or by range (4XX response codes - one component - client errors, 5XX another component - server errors).
In this case 429 is very specific, and very different than 404, so I would give them each one component,
for 5xx type of response code we could have f.e. `ServerError`